docs/src/introduction.md

```diff
@@ -1,24 +1,39 @@
 
-> *This documentation is still Work In Progress. Some topics have been discussed in greater depth, while others require additional clarification. Sections of this documentation might later be reorganized in order to achieve a better flow.*
-
-## Welcome to the Polygon Miden Documentation
-Polygon Miden is a zkRollup for high-throughput and private applications. Users can create zero-knowledge proofs of their own state changes, and the Miden network verifies. It is expected to launch a public testnet in Q1 2024.
-
-This documentation explains how to develop on Miden. Furthermore, it explains the Miden architecture and concepts, the network and its components, and the underlying cryptographic primitives.
+Polygon Miden is a modular execution layer that extends Ethereum's capabilities using powerful features such as parallel transaction execution and client-side proving. With Miden, developers can create novel, high-throughput, privacy preserving dApps for DeFi, RWA and Autonomous Worlds using their favorite languages such as Rust and TypeScript.
 
 If you want to join the technical discussion, please check out
 
 * the [Discord](https://discord.gg/0xpolygondevs)
 * the [Repo](https://github.com/0xPolygonMiden)
 * the [Roadmap](roadmap.md)  
 
-## Miden creates a new design space secured by Ethereum
-Our goal is to not only scale Ethereum but to extend it. Rollups - secured by Ethereum - can be new design spaces and even experimental. This is the place to innovate. The base layer, however, should stay conservative and only slowly evolve to ensure the required safety and stability.
+> *This documentation is still Work In Progress. Some topics have been discussed in greater depth, while others require additional clarification. Sections of this documentation might later be reorganized in order to achieve a better flow.*
+
+## Status and features
+Polygon Miden is currently on release v0.1. This is an early version of the protocol and its components. We expect to keep making even breaking changes to all components. 
+
+At this point, adventurous Pioneers can execute first transactions and send assets to each other. Polygon Miden doesn't offer all the features one would expect from a zkRollup, yet. During 2024, we expect to offer gradually more features. Eventually, developers should be able to code any application they want on Polygon Miden.
 
-Like other rollups, we scale Ethereum and inherit its security. We want to provide a safe and decentralized environment for composable smart contracts.
+### Feature highlights
+* **Private accounts**. The Miden Operator only tracks a commitment to any account data in the public database. Users can only execute smart contracts of which they know the interface. 
+* **Private notes**. Like private accounts, the Miden Operator only tracks a commitment to any notes in the public database. Users need to submit the note data off-chain to each other in order to consume private note in transactions.
```

Comment:
Potentially better wording for "Users need to submit...": Users need to communicate note details to each other off-chain (via any side channel) in order to consume private notes in transactions.